defs/bridge2.cue

```diff
@@ -13,7 +13,9 @@ services: "bridge2": {
     environment: {
       BRIDGE_TRANSCRIBE_URL: "http://substrate:8080/faster-whisper/v1/transcribe"
       BRIDGE_TRANSLATE_URL: "http://substrate:8080/seamlessm4t/v1/transcribe"
-      BRIDGE_ASSISTANT_URL: "http://substrate:8080/airoboros-l2-13b-2.2/v1/completions"
+      // BRIDGE_ASSISTANT_URL: "http://substrate:8080/airoboros-l2-13b-2.2/v1/completions"
+      BRIDGE_ASSISTANT_URL: "http://substrate:8080/llama-3-8b-instruct-awq/v1/completions"
```

Comment:
@ajbouh it seems like llama-3 will consume basically all of the available GPU memory, so I started getting OOM errors for airoboros (some of the allocations only seemed to be for like 20M, though I don't know if it needed more in total). Do you have thoughts on configuring it to keep both, or just run everything through llama-3?

Reply:
Being able to balance all of these different models at once is not easy. This is a thing I was working on handling automatically. Using llama-3 for everything is probably easiest until we can handle eviction properly. Also getting multiple cards going would help this.